code/color_space/src/viewmodel/view_model.py

```diff
@@ -9,7 +9,7 @@
 class ViewModel:
     @staticmethod
     def is_correct(color):
-        return map(operator.methodcaller("isdigit"), color) == [True] * 3
+        return list(map(operator.methodcaller("isdigit"), color)) == [True] * 3
```

Comment:
the whole line looks suspicious, why not `return all([isdigit(x) for x in color])` or something similar?

Reply:
@okainov well, this is your line, i've just fixed it a bit ;-) Feel free to submit a PR, will be able to return to it a bit later...